src/h5web/toolbar/controls/DomainSlider/Histogram.tsx

```diff
@@ -52,13 +58,20 @@ function Histogram(props: Props) {
             x={xScale(bins[i])}
             y={height - yScale(d)}
             width={xScale(bins[i + 1]) - xScale(bins[i]) + 0.5} // +0.5 removes the small gap between bars
-            height={yScale(d)}
+            height={yScale(d) - yScale(0)}
           />
         ))}
-        <HistogramIndicators
-          positions={sliderDomain.map(xScale) as Domain}
-          height={height}
-        />
+        {colorMap && (
+          <ColorBarGradient
+            x={indicatorPositions[0]}
+            y={height - yScale(0)}
+            width={indicatorPositions[1] - indicatorPositions[0]}
+            height={yScale(0) - yScale(yMin)}
```

Comment:
Placing the colorbar between the indicators in X and between `0` and `yMin` in Y